lib/workers/miq_worker_types.rb

```diff
@@ -29,13 +29,15 @@
   "ManageIQ::Providers::Kubernetes::ContainerManager::EventCatcher"           => %i(manageiq_default),
   "ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCollectorWorker" => %i(manageiq_default),
   "ManageIQ::Providers::Kubernetes::ContainerManager::RefreshWorker"          => %i(manageiq_default),
+  "ManageIQ::Providers::Kubernetes::MonitoringManager::EventCatcher"          => %i(kubernetes openshift),
```

Comment:
@jrafanie what's the impact of using `kubernetes openshift` here for the mapping?  I see everything else uses `manageiq_default`.

Reply:
@blomquisg Currently, we have tight coupling to the manageiq-api and vmware provider gems, see [here](https://github.com/ManageIQ/manageiq/pull/15821#issuecomment-322812756), so you'll get NameError/LoadError when core code tries to load Api and vmware.  For now, we need to use manageiq_default here to be safe.

Reply:
Thanks, updated